code/modules/unit_tests/species_change_clothing.dm

```diff
@@ -0,0 +1,16 @@
+///Gives a Human lizard-incompatible shoes, then changes their species over to see if they drop the now incompatible shoes.
+/datum/unit_test/species_change_clothing
+
+/datum/unit_test/species_change_clothing/Run()
+	// Test lizards as their own thing so we can get more coverage on their features
+	var/mob/living/carbon/human/human_to_lizard = allocate(/mob/living/carbon/human/dummy/consistent)
+	human_to_lizard.equipOutfit(/datum/outfit/job/assistant/consistent)
+	human_to_lizard.dna.features["legs"] = DIGITIGRADE_LEGS //you WILL have digitigrade legs
```

Comment:
you could just set them to an ashie, ashwalkers always have digi legs.

Reply:
Refer to Melbert's review